react-training/src/redux/auth/action.js

```diff
@@ -4,39 +4,43 @@ import { getByEmail } from '../../services/loginService';
 export const actionTypes = {
```

Comment:
Let's use [createTypes](https://github.com/Wolox/redux-recompose/blob/master/src/creators/createTypes/docs.md) with [completeTypes](https://github.com/Wolox/redux-recompose/blob/master/src/completers/completeTypes/docs.md) for api integration actions